sycl/test/basic_tests/get_nonhost_devices.cpp

```diff
@@ -1,6 +1,9 @@
-// RUN: %clangxx -fsycl %s -o %t.out
+// REQUIRES: opencl
```

Comment:
We can add cuda here,

Reply:
We have seen this test fail on machines that do not have OpenCL drivers but only CUDA installed. One fix might be to add `env SYCL_BE=%sycl_be` to the run line but we need to test this to see if it works. Adding the `%GPU_RUN_PLACEHOLDER` fixed the test, too.

Reply:
Changed.